package_manifest.yaml

```diff
@@ -164,6 +165,7 @@ pulpcore_packages:
     python-pulpcore-client: {}
     python-pycairo: {}
     python-pygobject: {}
+    python-pyopenssl: {}
```

Comment:
we have python-pyOpenSSL -- silly naming but that's what it is ;)

Reply:
I added this while rebasing our branch rpm/3.16 using rpm/3.15 as base